GenderPayGap.WebUI/Classes/Presentation/SubmissionService.cs

```diff
@@ -484,51 +481,12 @@ public async Task<ReturnViewModel> GetReturnViewModelAsync(long organisationId,
             return result;
         }
 
-        /// <summary>
-        ///     This draft will only return the contents of the draft, if you need to know the user that is currently locking the
-        ///     draft, or when was the draft last written, please use GetReportInfoModelWithDraft
-        /// </summary>
-        /// <param name="organisation"></param>
-        /// <param name="returnModifiedDate"></param>
-        /// <param name="snapshotYear"></param>
-        public async Task<ReportInfoModel> GetReportInfoModelWithLockedDraftAsync(Organisation organisation,
-            DateTime? returnModifiedDate,
-            int snapshotYear)
-        {
-            ReportInfoModel reportInfo = await GetReport_InfoAsync(organisation, returnModifiedDate, snapshotYear);
-            reportInfo.Draft = await GetDraftIfAvailableAsync(organisation.OrganisationId, snapshotYear);
-            return reportInfo;
-        }
-
         public DateTime GetCurrentSnapshotDate(SectorTypes sector = SectorTypes.Private)
         {
             // Get the current reporting period for the sector
             return GetSnapshotDate(sector);
         }
 
-        public async Task<List<ReportInfoModel>> GetAllEditableReportsAsync(UserOrganisation userOrg, DateTime currentSnapshotDate)
-        {
-            int currentYear = currentSnapshotDate.Year;
-            int startYear = currentYear - (Global.EditableReportCount - 1);
-            if (startYear < Global.FirstReportingYear)
-            {
-                startYear = Global.FirstReportingYear;
-            }
-
-            var reportInfos = new List<ReportInfoModel>();
-            for (int snapshotYear = startYear; snapshotYear <= currentYear; snapshotYear++)
-            {
-                Return report = await GetReturnFromDatabaseAsync(userOrg.OrganisationId, snapshotYear);
-                ReportInfoModel reportInfo = report != null
-                    ? await GetReportInfoModelWithLockedDraftAsync(report.Organisation, report.Modified, snapshotYear)
-                    : await GetReportInfoModelWithLockedDraftAsync(userOrg.Organisation, null, snapshotYear);
-
-                reportInfos.Add(reportInfo);
-            }
-
-            return reportInfos;
-        }
-

```

Comment:
This is just a regular unused method (I think it became unused when we refactored something)